src/sentry/static/sentry/app/components/radio.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import classNames from 'classnames';
+
+const Radio = React.createClass({
```

Comment:
Can we refactor this so that it extends from `FormField`, similar to the components found in `components/forms/`

Reply:
i can make a radioButtonsField analogous to a the multipleCheckboxField, but i'm not sure i can use it here

Reply:
Let's just leave it for now then